src/V3Broken.cpp

```diff
@@ -166,7 +166,8 @@ class BrokenCheckVisitor final : public VNVisitorConst {
         nodep->brokenState(m_brokenCntCurrentUnder);
         const char* const whyp = nodep->brokenGen();
         UASSERT_OBJ(!whyp, nodep,
-                    "Broken link in node (or something without maybePointedTo): " << whyp);
+                    "Broken link in node (or something without maybePointedTo): " << whyp << " -- "
+                                                                                  << nodep);
```

Comment:
With --debug it always prints the node directly above anyways so this can be backed out.